docker/start-service.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+#------------ Usage details for shell script starts here ----------------
+usage()
+{
+  echo
+  echo "Usage: $0 -s -k -h"
+  echo
+  echo "    -s start feeder service"
+  echo "    -k stop feeder service"
+  echo "    -h this text"
+  echo
+}
+
+while getopts ':skh' opt; do
+    case $opt in
+        s)  START_SERVICE=1
+            ;;
+        k)  STOP_SERVICE=1
+            ;;
+        h)  usage
+            exit 1
+            ;;
+    esac
+done
+#------------ Usage details for shell script ends here ----------------
+
+#------------- Code to start feeder service starts here ---------------
+if [[ ${START_SERVICE} -eq "1" ]]; then
+  echo "Starting feeder service and postgres db..."
+  docker-compose up -d
+
+  FEEDER_APP_CONTAINER_ID=$(docker ps --filter "name=feederapp-container" -q)
+  FEEDER_DB_CONTAINER_ID=$(docker ps --filter "name=feederdb-container" -q)
+
+  if [[ -z ${FEEDER_DB_CONTAINER_ID} ]]; then
+    echo "postgres db is not running"
+    exit 5
+  fi
+
+  if [[ -z ${FEEDER_APP_CONTAINER_ID} ]]; then
+    echo "feeder service is not running"
+    docker-compose down
+    exit 5
+  fi
+
+  echo "Migrating tables to postgres db..."
+  docker exec $FEEDER_APP_CONTAINER_ID /bin/sh -c "bundle exec rake db:migrate"
+
+  echo "Starting webpack service..."
+  docker exec -d $FEEDER_APP_CONTAINER_ID /bin/sh -c "./bin/webpack-dev-server"
```

Comment:
It would be nice to define webpack-dev-server as another docker-compose service to simplify launch sequence. But that's ok, I'll do this later.

Reply:
@dreikanter i tried that, and yes i was able to run the webpack-dev-server separately. But, the issue is that the webpack server bundles the frontend related files and serves it to the rails app, but it had no context of the rails app when i started it as they were different processes. 
I'm not sure if I ran the webpack-dev-server after I ran the rails app. Need to check that.

Reply:
I'll merge this patch as is, and will think about running WDS as docker-compose service later.

BTW, if you're interested or need another PR for Hacktoberfest challenge, feel free to participate :) I didn't research this subject closely yet, but here are some examples: https://www.plymouthsoftware.com/articles/dockerising-webpacker, https://github.com/rails/webpacker/blob/master/docs/docker.md